### ISSUE TITLE
Discrepancy Between SMSE Formula in Paper and Code Implementation

### ISSUE THREAD

**Issue by sifanexisted** [pdearena/pdearena#41]
Dear authors,

First of all, congratulations on these works; they are a timely contribution to scientific machine learning! However, I found that the computed rollout metrics in the code are not consistent with the formula in the paper.

In the paper, the SMSE is given by:

![image](https://github.com/pdearena/pdearena/assets/70182613/f9e050c2-ea28-49f5-a74d-0cebf494a9dd)

However, from [code](https://github.com/pdearena/pdearena/blob/22360a766387c3995220b4a1265a936ab9a81b88/pdearena/models/pdemodel.py#L318), it seems to compute the sum over the time dimension but average over different channels (axis=3), as the prediction has a shape of B T C H W.

```
pred_traj = rollout2d(
                self.model,
                init_u,
                init_v,
                grid,
                self.pde,
                self.hparams.time_history,
                self.hparams.max_num_steps,
            )
            targ_u = u[:, target_start_time:target_end_time, ...]
            if self.pde.n_vector_components > 0:
                targ_v = v[:, target_start_time:target_end_time, ...]
                targ_traj = torch.cat((targ_u, targ_v), dim=2)
            else:
                targ_traj = targ_u
            loss = self.rollout_criterion(pred_traj, targ_traj).mean(dim=(0, 2, 3, 4))
```

Could you please clarify this discrepancy?

Thank you.
